test/e2e/bootstrapper_test.go

```diff
@@ -89,6 +89,14 @@ func TestBootstrapper(t *testing.T) {
 		time.Sleep(2 * time.Second)
 		assert.Truef(t, svcRunning(t, bootstrapper.KubeletServiceName), "The kubelet service is not running")
 	})
+
+	// Kubelet arguments with paths that are set by bootstrapper
+	// Does not include node-labels and container-image since their paths do not depend on underlying OS
+	checkPathsFor := []string{"--bootstrap-kubeconfig", "--cloud-config", "--config", "--kubeconfig", "--log-file",
```

Comment:
Why do you want to create `checkPathsFor` here and pass it to testing function, why can't you create this in the callee?

Reply:
The callee function is called by both `TestBootstrapper` and `testConfigureCNI`. So one is a test for `initialize-kubelet` and the other is for `configure-cni`. Hence the kubelet arguments present for `configure-cni` would include cni specific arguments, which won't be present for `initialize-kubelet`. So I am changing this variable specific to the test case before calling the func `testPathInKubeletArgs`